src/v2/view-builder/utils/IdpUtil.js

```diff
@@ -0,0 +1,31 @@
+import { FORMS } from 'v2/ion/RemediationConstants';
+
+export function getIdvName(remediations) {
+  let redirectIDVerifyRemediation = remediations.find(remediation => {
+    return remediation.name === FORMS.REDIRECT_IDVERIFY;
+  });
+  return redirectIDVerifyRemediation.idp.name;
+}
+
+export function getHelpLinks(remediations) {
+  const idpName = getIdvName(remediations);
+  let termsOfUse, privacyPolicy;
+  switch (idpName?.toUpperCase()) {
+  case 'PERSONA':
+    termsOfUse = 'https://withpersona.com/legal/terms-of-use';
+    privacyPolicy = 'https://withpersona.com/legal/privacy-policy';
+    break;
+  case 'CLEAR':
+    termsOfUse = 'https://www.clearme.com/member-terms';
+    privacyPolicy = 'https://www.clearme.com/privacy-policy';
+    break;
+  case 'INCODE':
+    termsOfUse = 'https://incode.id/terms';
+    privacyPolicy = 'https://incode.id/privacy';
+    break;
+  }
+  return {
```

Comment:
See my comment in the equivalent gen3 code about standardizing on a `default` case that explicitly returns `undefined` for both `termsOfUse` and `privacyPolicy`